wweval/R/supplement_analyses_and_figs.R

```diff
@@ -0,0 +1,728 @@
+get_plot_scores_and_forecasts <- function(scores_single_loc_date,
+                                          eval_output_subdir,
+                                          n_calib_days = 10) {
+  this_location <- scores_single_loc_date |>
+    dplyr::distinct(location) |>
+    dplyr::pull()
+
+  this_forecast_date <- scores_single_loc_date |>
+    dplyr::distinct(forecast_date) |>
+    dplyr::pull()
+  colors <- plot_components()
+
+
+  ## Get the forecasts from file storage
+  fp_ww <- get_filepath(
+    eval_output_subdir,
+    scenario = "status_quo",
+    forecast_date = this_forecast_date,
+    model_type = "ww",
+    location = this_location,
+    output_type = glue::glue("hosp_quantiles"),
+    file_extension = "tsv"
+  )
+
+  if (file.exists(fp_ww)) {
+    hosp_quantiles_ww <- readr::read_tsv(fp_ww)
+
+    fp_hosp <- get_filepath(
+      eval_output_subdir,
+      scenario = "no_wastewater",
+      forecast_date = this_forecast_date,
+      model_type = "hosp",
+      location = this_location,
+      output_type = glue::glue("quantiles"),
+      file_extension = "tsv"
+    )
+    hosp_quantiles_hosp <- readr::read_tsv(fp_hosp)
+
+
+    min_scores_date <- min(scores_single_loc_date$date)
+
+    quantiles <- hosp_quantiles_ww |>
+      dplyr::bind_rows(hosp_quantiles_hosp)
+
+    quantiles_wide <- quantiles |>
+      dplyr::filter(
+        quantile %in% c(0.025, 0.25, 0.5, 0.75, 0.975)
+      ) |>
+      tidyr::pivot_wider(
+        id_cols = c(
+          location, forecast_date, period, scenario,
+          date, t, eval_data, calib_data, model_type
+        ),
+        names_from = quantile,
+        values_from = value
+      )
+
+    quantiles_wide_forecast <- quantiles_wide |>
+      dplyr::filter(date >= min_scores_date)
+
+
+    scores_avg <- scores_single_loc_date |>
+      dplyr::group_by(forecast_date, location, model) |>
+      dplyr::summarize(avg_crps = mean(crps))
+
+    p_scores_t <- ggplot(scores_single_loc_date) +
+      geom_line(aes(x = date, y = crps, color = model),
+        show.legend = FALSE
+      ) +
+      get_plot_theme(
+        x_axis_dates = TRUE
+      ) +
+      scale_x_date(
+        date_breaks = "1 week",
+        labels = scales::date_format("%Y-%m-%d"),
+      ) +
+      xlab("") +
```

Comment:
```suggestion
      xlab(NULL) +
```

subtly different. `xlab("")` leaves vertical space where the label would go. `xlab(NULL)` erases the vertical space

Reply:
This occurs several times. Recommend find and raplacing.